tests/handlers/test_profile.py

```diff
@@ -75,7 +75,7 @@ def register_query_handler(query_type, handler):
     @defer.inlineCallbacks
     def test_get_my_name(self):
         yield self.store.set_profile_displayname(
-            self.frank.localpart, "Frank"
+            self.frank.localpart, "Frank", 1
```

Comment:
it'd be nice to add trailing commas while you're here

Reply:
ah yep, done (plus the others)